src/screens/FeedScreen/Content.js

```diff
@@ -41,7 +41,8 @@ const Content = ({
   const route = useRoute();
   const [textHeight, setTextHeight] = React.useState(null);
   const maxFontSize = normalizeFontSizeByWidth(28);
-  const minFontSize = normalizeFontSizeByWidth(22);
+  const minFontSize = normalizeFontSizeByWidth(16);
+  const [layoutHeight, setLayoutHeight] = React.useState(null);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The minimum font size has been reduced from 22 to 16, and a new state variable `layoutHeight` has been introduced. Ensure that these changes do not negatively impact the readability of the text on different screen sizes.
```diff
- const minFontSize = normalizeFontSizeByWidth(22);
+ const minFontSize = normalizeFontSizeByWidth(16);
+ const [layoutHeight, setLayoutHeight] = React.useState(null);
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->